torch/csrc/jit/codegen/cuda/scheduler/utils.h

```diff
@@ -360,16 +360,6 @@ struct BroadcastMultipleInformation {
 TORCH_CUDA_CU_API BroadcastMultipleInformation
 getBroadcastMultiples(TensorView* reference_tv, DataType index_type);
 
-//! Collect maximum vectorization word size of a tensor whose
-//! innermost domain is leaf_merged_domain. Contig merging is taken
-//! into account to expand vectorization if possible.
-size_t collectMaxVectorizeSizeWithContigMerge(
```

Comment:
Is this function removed? Why is it only removed in the header file?

Reply:
Just not used anywhere but inside vectorize_helper, put a namespace around it in that file to be more clear.

Reply:
For some reason it was removed in the benchmark branch, so was following through on that.